From d2d9ac6aae1b743d729b2e4027d5666b2bc93003 Mon Sep 17 00:00:00 2001
From: David Sterba <dsterba@suse.com>
Date: Fri, 23 Sep 2016 13:54:09 +0200
Subject: btrfs: tests: constify free space extent specs

We don't change the given extent ranges, mark them const to catch
accidental changes.

Signed-off-by: David Sterba <dsterba@suse.com>
---
 fs/btrfs/tests/free-space-tree-tests.c | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/fs/btrfs/tests/free-space-tree-tests.c b/fs/btrfs/tests/free-space-tree-tests.c
index c449da4..061b4d5 100644
--- a/fs/btrfs/tests/free-space-tree-tests.c
+++ b/fs/btrfs/tests/free-space-tree-tests.c
@@ -31,7 +31,7 @@ static int __check_free_space_extents(struct btrfs_trans_handle *trans,
 				      struct btrfs_fs_info *fs_info,
 				      struct btrfs_block_group_cache *cache,
 				      struct btrfs_path *path,
-				      struct free_space_extent *extents,
+				      const struct free_space_extent * const extents,
 				      unsigned int num_extents)
 {
 	struct btrfs_free_space_info *info;
@@ -120,7 +120,7 @@ static int check_free_space_extents(struct btrfs_trans_handle *trans,
 				    struct btrfs_fs_info *fs_info,
 				    struct btrfs_block_group_cache *cache,
 				    struct btrfs_path *path,
-				    struct free_space_extent *extents,
+				    const struct free_space_extent * const extents,
 				    unsigned int num_extents)
 {
 	struct btrfs_free_space_info *info;
@@ -165,7 +165,7 @@ static int test_empty_block_group(struct btrfs_trans_handle *trans,
 				  struct btrfs_path *path,
 				  u32 alignment)
 {
-	struct free_space_extent extents[] = {
+	const struct free_space_extent extents[] = {
 		{cache->key.objectid, cache->key.offset},
 	};
 
@@ -179,7 +179,7 @@ static int test_remove_all(struct btrfs_trans_handle *trans,
 			   struct btrfs_path *path,
 			   u32 alignment)
 {
-	struct free_space_extent extents[] = {};
+	const struct free_space_extent extents[] = {};
 	int ret;
 
 	ret = __remove_from_free_space_tree(trans, fs_info, cache, path,
@@ -200,7 +200,7 @@ static int test_remove_beginning(struct btrfs_trans_handle *trans,
 				 struct btrfs_path *path,
 				 u32 alignment)
 {
-	struct free_space_extent extents[] = {
+	const struct free_space_extent extents[] = {
 		{cache->key.objectid + alignment,
 			cache->key.offset - alignment},
 	};
@@ -224,7 +224,7 @@ static int test_remove_end(struct btrfs_trans_handle *trans,
 			   struct btrfs_path *path,
 			   u32 alignment)
 {
-	struct free_space_extent extents[] = {
+	const struct free_space_extent extents[] = {
 		{cache->key.objectid, cache->key.offset - alignment},
 	};
 	int ret;
@@ -248,7 +248,7 @@ static int test_remove_middle(struct btrfs_trans_handle *trans,
 			      struct btrfs_path *path,
 			      u32 alignment)
 {
-	struct free_space_extent extents[] = {
+	const struct free_space_extent extents[] = {
 		{cache->key.objectid, alignment},
 		{cache->key.objectid + 2 * alignment,
 			cache->key.offset - 2 * alignment},
@@ -273,7 +273,7 @@ static int test_merge_left(struct btrfs_trans_handle *trans,
 			   struct btrfs_path *path,
 			   u32 alignment)
 {
-	struct free_space_extent extents[] = {
+	const struct free_space_extent extents[] = {
 		{cache->key.objectid, 2 * alignment},
 	};
 	int ret;
@@ -311,7 +311,7 @@ static int test_merge_right(struct btrfs_trans_handle *trans,
 			   struct btrfs_path *path,
 			   u32 alignment)
 {
-	struct free_space_extent extents[] = {
+	const struct free_space_extent extents[] = {
 		{cache->key.objectid + alignment, 2 * alignment},
 	};
 	int ret;
@@ -350,7 +350,7 @@ static int test_merge_both(struct btrfs_trans_handle *trans,
 			   struct btrfs_path *path,
 			   u32 alignment)
 {
-	struct free_space_extent extents[] = {
+	const struct free_space_extent extents[] = {
 		{cache->key.objectid, 3 * alignment},
 	};
 	int ret;
@@ -396,7 +396,7 @@ static int test_merge_none(struct btrfs_trans_handle *trans,
 			   struct btrfs_path *path,
 			   u32 alignment)
 {
-	struct free_space_extent extents[] = {
+	const struct free_space_extent extents[] = {
 		{cache->key.objectid, alignment},
 		{cache->key.objectid + 2 * alignment, alignment},
 		{cache->key.objectid + 4 * alignment, alignment},
-- 
cgit v0.12

